interface/portraits/portraits_tauren.gfx

```diff
@@ -23,15 +33,18 @@ spriteTypes = {
 
 		hair_color_index = 8
 		hair_color = { # dark, base, highlight
-			{ 36 35 29 } { 196 172 91 } { 255 255 255 }
+			{ 0 0 0 } {  33 33 33 } { 255 255 255 } # Black
+			{ 0 0 0 } {  63 34 34 } { 255 255 255 } # Dark Brown
+			{ 0 0 0 } {  55 33 0 } { 255 255 255 } # Red Dark Brown
 		}
 
 		eye_color_index = 9
 		eye_color = {
-			{ 58 109 193}
+			{ 151 44 66}
+			{ 32 109 45}				
 		}
```

Comment:
It doesn't work because you don't have an iris layer (this layer is gray and coloured to this colour).
Check how the dwarves are done.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105495505-b33f6d80-5cd5-11eb-9123-87d88bc4c88b.png)

</details>